sys/include/sema.h

```diff
@@ -33,23 +35,35 @@ extern "C" {
 /**
  * @brief   Creates semaphore statically.
  *
- * @param[in] value Initial value for the semaphore.
+ * @param[in] value Initial value for the semaphore (can't be 0). For a 0
+ *                  initialized semaphore @see SEMA_CREATE_EMPTY
```

Comment:
See https://github.com/RIOT-OS/RIOT/pull/6155#discussion_r92351261 the restriction is that when the semaphore value is 0 the mutex needs to be locked.